library/sbi-rt/src/fwft.rs

```diff
@@ -0,0 +1,57 @@
+//! Chapter 18. SBI Firmware Features Extension (EID #0x46574654 "FWFT")
```

Comment:
```suggestion
//! Chapter 18. SBI Firmware Features Extension (EID #0x46574654 "FWFT").
```

Module-level introduction document must ends with `.` to form a complete sentence.